v0/destinations/indicative/data/INPageConfig.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "destKey": "eventUniqueId",
+    "sourceKeys": [
+      "userId",
+      "context.traits.userId",
+      "context.traits.id",
+      "anonymousId"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "eventName",
+    "sourceKeys": "event",
```

Comment:
page doesn't have event correct? should be name?

Reply:
Updated